client/http/client.go

```diff
@@ -154,7 +159,7 @@ func (c *client) execDuration(name string, duration time.Duration) {
 // it consistent with the current implementation of some clients (e.g. TiDB).
 func (c *client) requestWithRetry(
```

Comment:
Is it possible to customize the retry?

Reply:
We will provide the function of custom backoffer in the future, but at present, the interface we are about to replace in TiDB uses this simple retry method.

Reply:
I still prefer to leave it to the caller even if the retry method is simple.

Reply:
Make sense to me though I'd like to implement it in an independent PR later.